x/wasm/keeper/proposal_handler.go

```diff
@@ -236,3 +248,30 @@ func handleUpdateInstantiateConfigProposal(ctx sdk.Context, k types.ContractOpsK
 	}
 	return nil
 }
+
+func handleUpdateDeploymentWhitelistProposal(ctx sdk.Context, k types.ContractOpsKeeper, p types.UpdateDeploymentWhitelistProposal) error {
+	if err := p.ValidateBasic(); err != nil {
+		return err
+	}
+
+	var config types.AccessConfig
+	if len(p.DistributorAddrs) == 0 {
```

Comment:
need clear config.address when AccessTypeNobody or AccessTypeEverybody

Reply:
config.Address is "" by default